ipatests/test_integration/util.py

```diff
@@ -86,5 +87,5 @@ def ldappasswd_user_change(user, oldpw, newpw, master):
     userdn = "uid={},{},{}".format(user, container_user, basedn)
 
     args = [paths.LDAPPASSWD, '-D', userdn, '-w', oldpw, '-a', oldpw,
-            '-s', newpw, '-x']
+            '-s', newpw, '-x', '-H', api.env.ldap_uri]
```

Comment:
Have you tested this?

This code is executed on controller machine, thus api.env may be undefined, because there is no installed IPA.
I'm not sure if ldap_uri may be diferent on each (I assume than ldapi URI is the same on all machines, but not ldaps)